src/connectors/sort-by/connectSortBy.ts

```diff
@@ -132,7 +132,14 @@ const connectSortBy: SortByConnector = function connectSortBy(
       dispose({ state }) {
         unmountFn();
 
-        return state.setIndex(connectorState.initialIndex || '');
+        warning(
+          connectorState.initialIndex !== undefined,
+          "The index name can't be undefined when unmounting."
+        );
```

Comment:
I don't think this really needs a warning, since there's no practical way of it happening :)

Reply:
Should it need a test then? As it can't really happen